codecoverage.py

```diff
@@ -104,7 +111,7 @@ def _is_chosen_task(t):
             if not any(suite in t['task']['metadata']['name'] for t in test_tasks):
                 warnings.warn('Suite %s not found' % suite)
 
-    for test_task in test_tasks:
+    for test_task in test_tasks[:1]:
```

Comment:
I assume this was for testing, it should be removed now :smiley:

Reply:
Oops, yes. :)